packages/reports/src/Components/Elements/Modals/MarketUnavailableModal/__tests__/market-unavailable.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render, screen } from '@testing-library/react';
+import MarketUnavailableModal from '../market-unavailable';
+import { mockStore } from '@deriv/stores';
+import ReportsProviders from '../../../../../reports-providers';
+
+const mock_props = {
+    onCancel: jest.fn(),
+    onConfirm: jest.fn(),
+};
+
+describe('MarketUnavailableModal', () => {
+    it('should render modal component', () => {
+        const mock_root_store = mockStore({ ui: { has_only_forward_starting_contracts: true } });
+
+        (ReactDOM.createPortal as jest.Mock) = jest.fn(component => {
```

Comment:
Do we need to clean it?
`(ReactDOM.createPortal as jest.Mock).mockClear();`
Please, ignore if irrelevant

Reply:
I have no idea, I copied the test case from dtrader package and it seems you wrote it yourself 😄 I'd guess there was no need to clear it previously, so let's not add any inconsistencies between the packages.

Reply:
In most cases when I observe my own old code it make me embarrassed )
- Who wrote it?! Oh, okay, it was me